kedro/templates/project/{{ cookiecutter.repo_name }}/README.md

```diff
@@ -51,17 +51,17 @@ To generate or update the dependency requirements for your project:
 kedro build-reqs
 ```
 
-This will `pip-compile` the contents of `src/requirements.txt` into a new file `src/requirements.lock`. You can see the output of the resolution by opening `src/requirements.lock`.
+This will `pip-compile` the contents of `requirements.txt` into a new file `requirements.lock`. You can see the output of the resolution by opening `requirements.lock`.
```

Comment:
I noticed #3065 removed the deprecated CLI commands and #3072 removed the requirements, but we're still mentioning `kedro build-reqs` here. Was there a PR to remove mentions from the docs and this was an oversight? Or is that task still pending? Sorry I lost track a bit 😓 @merelcht

Reply:
That's definitely oversight. I assumed it was removed with the CLI commands and docs, so I didn't do another check in my PR. Can do another one if I find some time today.

Reply:
This has been completed now.